.github/workflows/ci-for-diff-ns.yaml

```diff
@@ -44,6 +43,15 @@ jobs:
           kubectl rook_ceph -o test-operator -n test-cluster rook status
           kubectl rook_ceph -o test-operator -n test-cluster rook status all
           kubectl rook_ceph -o test-operator -n test-cluster rook status cephobjectstores
+          # use of sleep is not incouraged but here we need sleep between scaling down the osd and purging the osd
```

Comment:
In a future PR, can we refactor the script and take namespace as a var so we don't duplicate it?

Reply:
yes, I'll refactor the ci yaml's